packages/twenty-front/src/modules/action-menu/hooks/useRegisteredActions.ts

```diff
@@ -28,19 +26,26 @@ export const useRegisteredRecordActions = ({
     contextStoreTargetedRecordsRule,
   );
 
-  const actionConfig = getActionConfig(objectMetadataItem);
+  const recordActionConfig = getActionConfig(objectMetadataItem);
+
+  const recordAgnosticActionConfig = RECORD_AGNOSTIC_ACTIONS_CONFIG;
+
+  const actionsConfig = {
+    ...recordActionConfig,
+    ...recordAgnosticActionConfig,
+  };
 
   const actionsToRegister = isDefined(viewType)
-    ? Object.values(actionConfig ?? {}).filter(
+    ? Object.values(actionsConfig ?? {}).filter(
```

Comment:
nitpick: We don't need the nullish coalescing if we check if actionsConfig is defined

```suggestion
  const actionsToRegister = isDefined(viewType) && isDefined(actionsConfig)
    ? Object.values(actionsConfig).filter(
```